src/plugins/network.ts

```diff
@@ -50,14 +53,16 @@ export class NetworkManager {
     const isNavigationRequest = requestId == loaderId && type === 'Document';
 
     this.waterfallMap.set(requestId, {
+      step: this._currentStep,
+      timestamp: getTimestamp(),
       url,
       request,
       type,
       method,
-      start: timestamp,
+      requestSentTime: timestamp,
```

Comment:
This change was done after some investigation with the chrome network events and how we will use it in the waterfall. copy pasting from internal gist

So here is the gist that i found and we need to change

* we will use requestTime  for the waterfall stuff
* start/end represents the timestamp of when request is sent and when request completed, so we can calculate the queue time as follows
 ```js
const queuedTime = payload.response.timing.request_time - payload.response.start
if (queuedTime > 0) {
  // convert in ms and use it in waterfall
}
```
* we discard end timing for now as its no useful. Also i think we can rename it to something like requestStartTime  and responseEndTime to make it easier